src/db.js

```diff
@@ -3,33 +3,69 @@ const { Etcd3 } = require('etcd3');
 
 const etcd = new Etcd3({hosts: ETCD_URLS});
 
+const SERVICE_BASE_URL = "/services"
```

Comment:
@gutorc92, pinning everything on `/services` can be dangerous with regards to how the federation features work. Promster of higher federation levels (`gt 1`) looks at the `REGISTRY_ETCD_BASE` ENV to **list** all the Promsters available to federate. If everything is on `/services`, a level 2 Promster will consider everything behind `/services` to be a federation target.

Reply:
Promsters (gt 1) use the code below to set the SCRAPE_ETCD_PATH:

```
if [[ $ll -ne 0 ]]; then # if true, we properly configure bb-promster to behave like a level-n promster
    export SCRAPE_ETCD_PATH="${REGISTRY_ETCD_BASE}-promster-${SCRAPE_MATCH_REGEX}/${REGISTRY_SERVICE}" # configure where to look for federation targets
 ```
SCRAPE_ETCD_PATH will be set to "/services-promster-l{1, 2, 3, ...}". 
So, I exclude all registers from etcd that includes substring "promster"  listApps function:

```js
apps = apps.filter(app => !app.includes("promster") && app != "") 
```

Reply:
Nice @gutorc92! I didn't notice the code `apps = apps.filter(app => !app.includes("promster") && app != "")`